app/src/androidTest/java/com/simplemobiletools/calculator/MoneyActivityTest.java

```diff
@@ -75,7 +76,7 @@ public void deleteClearOnEmptyNumberTest() {
 
 
     @Test
-    public void taxWithoutGeoLocTest(){
+    public void taxWithoutGeoLocTest() {
```

Comment:
Not related to this pr but I like it. We should always include a space before bracket in a function definition.

Reply:
I agree good catch for styling

Reply:
Always space